core/amber/src/main/scala/edu/uci/ics/texera/web/model/request/ResultDownloadRequest.scala

```diff
@@ -0,0 +1,4 @@
+package edu.uci.ics.texera.web.model.request
+
+case class ResultDownloadRequest(downloadType: String, workflowName: String)
```

Comment:
I wonder what will happen if I submit two executions but share the same workflow name, will it be considered downloaded before because the request looks the same?

Reply:
The workflow name here is not the identifier. The request is transferred through session so the session id is the identifier. The workflow name here is used to set the title of the google sheet . I didn't find a way to retrieve the workflow name through the session id in the backend so I get it from the frontend. Duplicated names won't create any problems, as I tested already, since the session id for different workflow will be different.